<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mi Perfil</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../recursos.qrc">
    <normaloff>:/src/vista/imagenes/logoUniConecta.png</normaloff>:/src/vista/imagenes/logoUniConecta.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">

    QMainWindow {
        background-color: #f0f4f1;
    }
    QPushButton {
        background-color: #2e7d32;
        color: white;
        border-radius: 8px;
        font: bold 10pt &quot;Segoe UI&quot;;
        padding: 8px;
    }
    QPushButton:hover {
        background-color: #388e3c;
    }
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="mainLabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>600</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Century&quot;; color: #388e3c;</string>
    </property>
    <property name="text">
     <string>Perfil</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Usuario">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>90</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #d0e0ce;</string>
    </property>
    <property name="placeholderText" stdset="0">
     <string>Nombre de usuario</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Edad">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>130</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #d0e0ce;</string>
    </property>
    <property name="placeholderText" stdset="0">
     <string>Edad</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Descripcion">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>170</y>
      <width>400</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #d0e0ce;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Actividades">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>260</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #2e7d32; color: white; font-weight: bold; padding: 5px; border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>üìù Lista de Actividades</string>
    </property>
   </widget>
   <widget class="QLabel" name="ListaActividades">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>300</y>
      <width>400</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #d0e0ce;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BotonVolver">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>470</y>
      <width>150</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Volver</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../../recursos.qrc"/>
 </resources>
 <connections/>
</ui>
